### PR TITLE
Fix the crash with templating disabled

### PR DESCRIPTION
**Pull request by mvrhov**
DONE

### CODE REVIEW

**Comment by mvrhov** [comment]
@sanpii can you merge this and release 2.2.2 Thanks

File: sources/lib/Twig/Extension/ProfilerExtension.php [hunk 1]
```diff
@@ -30,9 +30,9 @@ class ProfilerExtension extends \Twig_Extension
      * Extension constructor.
      *
      * @access public
-     * @param  FilesystemLoader $loader
+     * @param  FilesystemLoader|\Twig_Loader_Filesystem $loader
      */
-    public function __construct(FilesystemLoader $loader)
```
Comment (sanpii) [inline sources/lib/Twig/Extension/ProfilerExtension.php]: You can use `\Twig_Loader_Filesystem` for type hinting, `FilesystemLoader` extends its.

Comment (sanpii) [inline sources/lib/Twig/Extension/ProfilerExtension.php]: Don’t forget to remove useless use.

Comment (mvrhov) [inline sources/lib/Twig/Extension/ProfilerExtension.php]: Oh I've missed that

**Comment by mvrhov** [comment]
done

File: sources/lib/Twig/Extension/ProfilerExtension.php [hunk 2]
```diff
@@ -30,9 +28,9 @@ class ProfilerExtension extends \Twig_Extension
      * Extension constructor.
      *
      * @access public
-     * @param  FilesystemLoader $loader
+     * @param  \Twig_Loader_Filesystem $loader
      */
-    public function __construct(FilesystemLoader $loader)
+    public function __construct($loader)
```
Comment (sanpii) [inline sources/lib/Twig/Extension/ProfilerExtension.php]: Type hinting is especially usefull in code :)

**Pull request merged**